src/components/layout/LeftMenu/LeftMenu.tsx

```diff
@@ -58,7 +59,9 @@ const LeftMenu = ({ regularLinks, pageSpecificLinks }: LeftMenuProps) => {
               icon={icon}
               buttonProps={{
                 ...buttonProps,
-                className: styles.link,
+                className: `${styles.link} ${
+                  router?.pathname === buttonProps.route ? styles.bold : ''
+                } ${buttonProps.className ?? ''}`,
```

Comment:
Let's use `classnames` to make it more readable since it looks overcomplicated
And let's move comparing to a variable `isActiveRoute`

`classnames(styles.link, buttonProps.className, { [styles.bold]: isActiveRoute })`